src/C2_economic_functions.py

```diff
@@ -123,8 +127,8 @@ def fuel_price_present_value(economics,):
     if economics["fuel_price_change_annual"] == 0:
         economics.update({"price_fuel": fuel_price_i})
     else:
-        for i in range(0, economics["project_lifetime"]):
-            cash_flow_fuel_l += fuel_price_i / (1 + economics["wacc"]) ** (i)
+        for i in range(0, economics[PROJECT_DURATION]):
```

Comment:
You need to import `DISCOUNTFACTOR` and `PROJECT_DURATION` from `src.constants_json_strings` at the top of the file